proposals/records.md

```diff
@@ -83,14 +58,58 @@ A record type contains two synthesized copying members if methods with the same
 signature are not already declared within the record type:
 
 * A protected constructor taking a single argument of the record type.
-* A public parameterless instance method called `Clone` which returns the record type.
+* A public parameter-less virtual instance "clone" method with a compiler-reserved name
 
 The protected constructor is referred to as the "copy constructor" and the synthesized
-body copies the values of all instance fields declared in the input type to the corresponding 
+body copies the values of all instance fields declared in the input type to the corresponding
 fields of `this`.
```

Comment:
>this`. [](start = 11, length = 6)

Just to be clear: I assume it is intentional that fields of the base are not copied. This will be extremely confusing to users.

Reply:
Neither intentional nor unintentional -- I see two viable specifications here. One way is to try to set up a pattern of delegation, where if the base type is a record, we call `base.Equals`, and the other where we always override and check all types regardless.

LDM hasn't decided on anything so I haven't spent time detailing either one.